src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/AggregateCompletionItemResolver.cs

```diff
@@ -44,15 +46,15 @@ public AggregateCompletionItemResolver(IEnumerable<CompletionItemResolver> compl
             }
         }
 
-        var resolvedCompletionItems = new Queue<VSInternalCompletionItem>();
+        using var _1 = ListPool<VSInternalCompletionItem>.GetPooledObject(out var resolvedCompletionItems);
```

Comment:
If you can be bothered, but unless I'm missing something it doesn't look like this needs to be a collection at all. A single `VSInternalCompletionItem? lastResolved` should do it.

Reply:
forest for the trees. Thank you :)